src/components/Table.jsx

```diff
@@ -0,0 +1,606 @@
+                }
+            })
+            .then(response => response.json())
+            .then(result => setBrands(result?.data?.uniqueValues.filter(item=> item !== null).map(item=>({brand:item}))) );
+    }, []);
+    useEffect(() => {
+        fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}&unique=assignee`, {
+                 method: "POST",
+                 headers: {
+                     "Content-type": "application/json; charset=UTF-8"
+                 }
+             })
+             .then(response => response.json())
+             .then(result => setAssignee(result?.data?.uniqueValues.filter(item=> item !== null)));
+     }, []);
+
+
+    useEffect(() => {
+        const body = {
+            "filters": {
+                "status":currentTab == "Unassigned" ? ["WAITING_FOR_WRITER"] : 
+                currentTab == "Completed"? ["WRITING_COMPLETE", "EDITING_COMPLETE"] : 
+                currentTab == "Assigned" ? ["ASSIGNED_TO_WRITER", "ASSIGNED_TO_EDITOR"] : 
+                currentTab == "InProgress" && ["WRITING_IN_PROGRESS", "EDITING_IN_PROGRESS"]
+            }
+        }
+        fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}&unique=status`, {
+                 method: "POST",
+                 body:JSON.stringify(body),
+                 headers: {
+                     "Content-type": "application/json; charset=UTF-8"
+                 }
+             })
+             .then(response => response.json())
+            .then(result => setStatus(result?.data?.uniqueValues));
+     }, [currentTab]);
+
+     useEffect(() => {
+        var date = new Date(searchByUpdatedAt);
+        const newDate = date.toDateString().split(' ')
+        const finalDate = `${newDate[3]}-${newDate[1]}-${newDate[2]}`
+        const newSelectedBrand = selectedBrand.map(item => item.brand)
+        const body = {
+            "filters": {
+                "styleId":searchByStyle,
+                "title":searchByTitle,
+                "brand":newSelectedBrand,
+                "lastUpdatedBy":searchByUpdatedBy,
+                "lastUpdateTs":finalDate,
+                "assignee":searchByAssignee,
+                "status": isStatusSelected ? [searchByStatus] : currentTab == "Unassigned" ? ["WAITING_FOR_WRITER"] : 
+                currentTab == "Completed"? ["WRITING_COMPLETE", "EDITING_COMPLETE"] : 
+                currentTab == "Assigned" ? ["ASSIGNED_TO_WRITER", "ASSIGNED_TO_EDITOR"] : 
+                currentTab == "InProgress" && ["WRITING_IN_PROGRESS", "EDITING_IN_PROGRESS"] 
+            },
+            "orderBy": {
+                "styleId": styleSort,
+                "title": titleSort,
+                "brand": brandSort,
+                "status": statusSort,
+                "assignee": assigneeSort,
+                "lastUpdatedBy": updatedBySort,
+                "lastUpdateTs":updatedAtSort
+            }
+        }
+        
+        searchByStyle == "" && delete body.filters.styleId
```

Comment:
As discussed in the call, dont follow this approach.
Build body if the values are present.
Don't add and delete like this.

Reply:
addressed